### ISSUE TITLE
Pyranges.subset drops Strand

### ISSUE THREAD

**Issue by tbrittoborges** [biocore-ntnu/pyranges#15]
First, thank you for your work. Alternatives to compute genomic arithmetics are always welcome!
```python
cs = pyranges.data.chipseq()

cs
+--------------+-----------+-----------+------------+-----------+--------------+
| Chromosome   | Start     | End       | Name       | Score     | Strand       |
| (category)   | (int32)   | (int32)   | (object)   | (int64)   | (category)   |
|--------------+-----------+-----------+------------+-----------+--------------|
| chr1         | 212609534 | 212609559 | U0         | 0         | +            |
| chr1         | 169887529 | 169887554 | U0         | 0         | +            |
| chr1         | 216711011 | 216711036 | U0         | 0         | +            |
| chr1         | 144227079 | 144227104 | U0         | 0         | +            |
| ...          | ...       | ...       | ...        | ...       | ...          |
| chrY         | 15224235  | 15224260  | U0         | 0         | -            |
| chrY         | 13517892  | 13517917  | U0         | 0         | -            |
| chrY         | 8010951   | 8010976   | U0         | 0         | -            |
| chrY         | 7405376   | 7405401   | U0         | 0         | -            |
+--------------+-----------+-----------+------------+-----------+--------------+
Stranded PyRanges object has 10,000 rows and 6 columns from 24 chromosomes.


def too_long(x):
    return (x.End - x.Start) < 120

cs.subset(too_long)

+--------------+-----------+-----------+------------+-----------+
| Chromosome   | Start     | End       | Name       | Score     |
| (category)   | (int32)   | (int32)   | (object)   | (int64)   |
|--------------+-----------+-----------+------------+-----------|
| chr1         | 212609534 | 212609559 | U0         | 0         |
| chr1         | 169887529 | 169887554 | U0         | 0         |
| chr1         | 216711011 | 216711036 | U0         | 0         |
| chr1         | 144227079 | 144227104 | U0         | 0         |
| ...          | ...       | ...       | ...        | ...       |
| chrY         | 15224235  | 15224260  | U0         | 0         |
| chrY         | 13517892  | 13517917  | U0         | 0         |
| chrY         | 8010951   | 8010976   | U0         | 0         |
| chrY         | 7405376   | 7405401   | U0         | 0         |
+--------------+-----------+-----------+------------+-----------+
Unstranded PyRanges object has 10,000 rows and 5 columns from 24 chromosomes.
```
Why do we miss the Strand column with the subset function? Is it a bug or intended behaviour?

**Comment by endrebak** [comment]
I fixed this (or tried to) in a recent version. What is your Pyranges
version? Does it happen if you try to get the latest version?

Thanks for reporting :)

Endre

**Comment by tbrittoborges** [comment]
Hi endre,

The one in pipy (version: 0.0.44). Tried to install from github and it was also pulling 0.0.44.

**Comment by endrebak** [comment]
Okay, I will fix it on Monday. It is probably fixed, but I have not pushed
the changes yet :)

**Comment by tbrittoborges** [comment]
Thanks, I will try it a little bit more.

**Comment by endrebak** [comment]
For the moment you can use apply to subset, but I will fix it ASAP

**Comment by endrebak** [comment]
Thanks for trying pyranges btw and giving feedback. I really appreciate it
:)

**Comment by endrebak** [comment]
Fixed in 0.0.45 out on PyPI. BioConda update is waiting for a review

**Comment by endrebak** [comment]
In 0.0.47 you can subset on rows using [] too:

```
import pyranges as pr

gr = pr.data.cpg()
print(gr[gr.CpG > 50])
```
